lifetimes/plotting.py

```diff
@@ -73,9 +73,9 @@ def plot_frequency_recency_matrix(model, max_x=None, max_t=None, **kwargs):
         max_t = int(model.data['T'].max())
 
     t = 1  # one unit of time
-    Z = np.zeros((max_t, max_x))
```

Comment:
This function can now use the `conditional_probability_alive_matrix` you defined above. Want to use that here?